src/test/resources/alma-fix/99371530278506441.json

```diff
@@ -101,7 +101,14 @@
   "exampleOfWork" : {
     "label" : "De ente et essentia",
     "id" : "https://d-nb.info/gnd/4117209-7",
-    "type" : [ "Work" ]
+    "type" : [ "Work" ],
+    "language" : [ {
+      "id" : "http://id.loc.gov/vocabulary/iso639-2/http://id.loc.gov/vocabulary/iso639-2/lat",
+      "label" : "Latein"
+    }, {
+      "id" : "http://id.loc.gov/vocabulary/iso639-2/http://id.loc.gov/vocabulary/iso639-2/rom",
```

Comment:
Here as well double language URI

Reply:
Also, where does the Romani come from? I have problems finding it in the [MARC version](https://github.com/hbz/lobid-resources/blob/1823-AddLanguageOfOriginal/src/test/resources/alma-fix/99371530278506441.xml)

Reply:
Sry, I messed up my testfile - "romani" was a manually created test case, as I didn't have a "real" MARC example. I forgot to create a new json after I reset the MARC file to the original state 
The double URI-prefix is now also correct.